tests/appsec/iast_aggregated_memcheck/test_aggregated_memleaks.py

```diff
@@ -1,8 +1,12 @@
+import pytest
+
 from tests.utils import override_env
 
 
-def test_aggregated_leaks():
-    with override_env({"DD_IAST_ENABLED": "True"}):
-        from scripts.iast.test_leak_functions import test_iast_leaks
+@pytest.mark.asyncio
```

Comment:
Just a blocking comment so it's not auto-merged because this tests doesn't seem to be running OK on CI (takes only one second, should be several minutes)